dts/arm/st/f4/stm32f4.dtsi

```diff
@@ -408,6 +408,15 @@
 			#io-channel-cells = <1>;
 		};
 
+		dac1: dac@40007400 {
```

Comment:
This peripheral is not present on all STM32F4 variants.
Please move this definition in following files:
stm32f405.dtsi, stm32f413.dtsi and stm32f446.dtsi.

Reply:
@erwango 
1: We don't have these boards, I can still raise git pull request with additions done for F405, F413 & F446 dtsi files, is it ok?
2: What about F429(ZI)? We have this board.

Reply:
1: Please do this inside current PR, this is fine.
2: Please have a look to inclusion scheme (429 includes 427, 427 includes ...)

Reply:
> 1: Please do this inside current PR, this is fine. => Added definition inside F405, F413 & F446 dtsi files. For F405, do we need to add new platform under boards/arm hierarchy?
For F413, the platform is nucleo_f413zh & for F446, the platform is nucleo_f446re. Please confirm.

> 2: Please have a look to inclusion scheme (429 includes 427, 427 includes ...) => Updated the dtsi code within inclusion.

Reply:
You don't need to enable DAC on those boards. The only point is to correctly enable DAC on .dtsi

Reply:
@erwango Corrected the definition changes for STM32 F429ZI platform.
Enabled DAC code on stm32f405.dtsi, stm32f413.dtsi and stm32f446.dtsi.